src/modules/CreateImage/components/CreateImageRoot.tsx

```diff
@@ -0,0 +1,22 @@
+import React, {PureComponent} from 'react';
+import CreateImageRootContainer from '../containers/CreateImageRootContainer';
+
+interface IProps {
+	step?: string;
+}
+
+const importsHash = {
+	initialization: <CreateImageRootContainer />,
+};
+
+export default class CreateImageRoot extends PureComponent<IProps> {
+	public static defaultProps = {
+		step: 'initialization',
+	};
+
+	public render() {
+		return (
+			importsHash[this.props.step || 'initialization']
```

Comment:
but we already have defaultProps step described. Why do we check for || ?

Reply:
it's just for mock, don't care about that